### PR TITLE
User can view their entry - Tiffany Brown

### PR DESCRIPTION
**Pull request by tiffanydbrown**
Screenshot 2023-11-16 at 1 35 08 PM

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -50,3 +60,28 @@ body {
...
+}
+
+li {
+  list-style-type: none;
+  font-size: 20px;
+}
+
+.hidden {
+  display: none;
+}
```
Comment (shawnkost) [inline css/styles.css]: This class is already provided for you in `layout.css`, so this isn't needed and can be deleted

File: js/main.js [hunk 2]
```diff
@@ -30,6 +30,85 @@ function handleSubmit(event) {
   $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
 
   $form.reset();
+
+  $ul.prepend(renderEntry(formValues));
+  viewSwap($entryView);
```
Comment (shawnkost) [inline js/main.js:35]: You should be passing in a string here, rather than a DOM object

File: js/main.js [hunk 3]
```diff
@@ -30,6 +30,85 @@ function handleSubmit(event) {
...
+
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+  $li.setAttribute('class', 'row');
+
+  const $picDiv = document.createElement('div');
+  $picDiv.setAttribute('class', 'column-half');
+
+  const $entryImage = document.createElement('img');
+  $entryImage.setAttribute('src', entry.photo);
```
Comment (shawnkost) [inline js/main.js]: Needs an `alt` attribute

File: js/main.js [hunk 4]
```diff
@@ -30,6 +30,85 @@ function handleSubmit(event) {
...
+}
+
+const $ul = document.querySelector('.ul');
+
+document.addEventListener('DOMContentLoaded', (event) => {
+  for (let i = 0; i < data.entries.length; i++) {
+    const $newEntry = renderEntry(data.entries[i]);
+    $ul.appendChild($newEntry);
+  }
+  viewSwap('entries');
```
Comment (shawnkost) [inline js/main.js]: You shouldn't be hard coding the string you pass in, you should be passing in `data.view`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Tiffany!

**Comment by tiffanydbrown** [comment]
Screenshot 2023-11-16 at 1 55 05 PM

File: js/main.js [hunk 5]
```diff
@@ -31,7 +31,7 @@ function handleSubmit(event) {
 
   $form.reset();
 
-  $ul.prepend(renderEntry(formValues));
+  $ul.prepend(renderEntry('#entry-form'));
```
Comment (shawnkost) [inline js/main.js]: Not the correct string, look at how you call this in other places

File: js/main.js [hunk 6]
```diff
@@ -74,7 +74,7 @@ document.addEventListener('DOMContentLoaded', (event) => {
     const $newEntry = renderEntry(data.entries[i]);
     $ul.appendChild($newEntry);
   }
-  viewSwap('entries');
+  viewSwap('data.view');
```
Comment (shawnkost) [inline js/main.js]: This shouldn't be a string. `data.view`s value is already a string

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple more changes here!

File: js/main.js [hunk 7]
```diff
@@ -31,7 +31,7 @@ function handleSubmit(event) {
 
   $form.reset();
 
-  $ul.prepend(renderEntry('#entry-form'));
+  $ul.prepend(renderEntry('$newEntry'));
```
Comment (shawnkost) [inline js/main.js]: Still not correct. What are the only two strings you are passing into `renderEntry`?

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change here Tiffany!

**Comment by tiffanydbrown** [comment]
Screenshot 2023-11-16 at 2 09 15 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**